tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -78,7 +78,7 @@ build:rbe --config=rbe_linux_cuda
 test:pycpp_filters --test_tag_filters=-no_oss,-oss_excluded,-oss_serial,-benchmark-test,-v1only,gpu,-no_gpu,-no_gpu_presubmit,-no_cuda11,-no_rocm
 test:pycpp_filters --build_tag_filters=-no_oss,-oss_excluded,-oss_serial,-benchmark-test,-v1only,gpu,-no_gpu,-no_gpu_presubmit,-no_cuda11,-no_rocm
 test:pycpp_filters --test_lang_filters=cc,py --test_size_filters=small,medium
-test:pycpp --config=pycpp_filters -- //tensorflow/... -//tensorflow/python/integration_testing/... -//tensorflow/compiler/tf2tensorrt/... -//tensorflow/core/tpu/... -//tensorflow/lite/... -//tensorflow/tools/toolchains/...
+test:pycpp --config=pycpp_filters -- //tensorflow/... -//tensorflow/python/integration_testing/... -//tensorflow/compiler/tf2tensorrt/... -//tensorflow/core/tpu/... -//tensorflow/lite/... -//tensorflow/tools/toolchains/... -//tensorflow/dtensor/python/tests:multi_client_test_2gpus -//tensorflow/dtensor/python/tests:multi_client_test_nccl_2gpus -//tensorflow/python/distribute/experimental:multi_worker_mirrored_strategy_test_2gpus
 
```

Comment:
hmmm....I'm not sure is a good idea to put bazelrc to skip these two. I think maybe should skip these three into https://github.com/ROCm/tensorflow-upstream/blob/develop-upstream/tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh 


cc @jayfurmanek

Reply:
I couldn't get it to work in the `if` block in the script.  I mentioned that in a meeting and @jayfurmanek mentioned to look into modifying the bazelrc file instead.